### ISSUE TITLE
Duotone not working in react-native

### ISSUE THREAD

**Issue by odesey** [FortAwesome/react-native-fontawesome#65]
**Describe the bug**
I've been using this package in an app I am working on for quite some time and it works great. I noticed that `Duotone` capability was finally implemented and I wanted to try it out. So I added it as per the instructions and tried the following:

```javascript
import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';

...
<FontAwesomeIcon icon="coffee" color="blue" secondaryColor="red" secondaryOpacity={ 0.4 } />
```

The icon rendered a solid blue with no duotone colors:

<img width="97" alt="Screen Shot 2020-06-04 at 1 04 52 PM" src="https://user-images.githubusercontent.com/3899366/83788968-0a6bd180-a664-11ea-8301-68df06774dd7.png">

**Expected behavior**
Icon should be rendered with duotone colors

**Additional context**
react-native: **0.61.5**
react-native-fontawesome:  **0.2.5**
react-native-svg: **12.1.0**
fontawesome-pro: **5.13.0**
fontawesome-svg-core: **1.2.28**
free-brands-svg-icons: **5.13.0**
free-solid-svg-icons: **5.13.0**
pro-duotone-svg-icons: **5.13.0**
pro-light-svg-icons: **5.13.0**
pro-regular-svg-icons: **5.13.0**
pro-solid-svg-icons: **5.13.0**

**Comment by robmadole** [comment]
Try this @odesey 

```jsx
<FontAwesomeIcon icon={['fad', 'coffee']} color="blue" secondaryColor="red" secondaryOpacity={ 0.4 } />```

**Comment by odesey** [comment]
Thanks for the quick response, I get the following with I try anything like

```javascript
 icon={['fad', 'coffee']} //no matter the combination it fails with the same error below
```

<img width="305" alt="Screen Shot 2020-06-04 at 1 48 57 PM" src="https://user-images.githubusercontent.com/3899366/83793320-620d3b80-a66a-11ea-940d-3dd52825f800.png">

**Comment by odesey** [comment]
Figured it out, this is what's needed:

```javascript

import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
import { faAbacus } from '@fortawesome//pro-duotone-svg-icons';

<FontAwesomeIcon
          icon={faAbacus}
          color="blue"
          secondaryColor="red"
          secondaryOpacity={0.4}
          size={64}
        />
```
I am not sure the `fad` option is being recognized in the `<FontAwesomeIcon icon={['fad', 'coffee']} />` component

**Comment by roy-law** [comment]
although the fad option works fine for me, but the duo-tone color is not supported

**Comment by pangpond** [comment]
It's work for me just upgrade to v0.2.5

```
import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
import {faSchool} from '@fortawesome/pro-duotone-svg-icons';

const HomeIcon = props => (
  <FontAwesomeIcon
    icon={faSchool}
    color="white"
    secondaryColor="white"
    secondaryOpacity={0.4}
  />
);

```

<img width="514" alt="Screen Shot 2563-07-22 at 14 43 44" src="https://user-images.githubusercontent.com/943742/88149107-d35c7a00-cc29-11ea-9b78-09074f6cb578.png">

**Comment by nustony** [comment]
I'm facing the same issue here. I want to use a dynamic name from a server like the code below but it doesn't work

```
import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';

...
<FontAwesomeIcon icon="coffee" color="blue" secondaryColor="red" secondaryOpacity={ 0.4 } />
```
Use like this, it works but is unable to make a dynamic name of icon: 
Any ideas?
`
import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
import { faAbacus } from '@fortawesome//pro-duotone-svg-icons';

<FontAwesomeIcon
          icon={faAbacus}
          color="blue"
          secondaryColor="red"
          secondaryOpacity={0.4}
          size={64}
        />`

**Comment by robmadole** [comment]
We've just released 0.3.0 with a lot of fixes. Please retest and I will re-open if this is still an issue for anyone.
